src/Sarif.Multitool.Library/SarifValidationContext.cs

```diff
@@ -96,6 +96,12 @@ public Uri TargetUri
 
         public DefaultTraces Traces { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum file size (in kilobytes) that will be analyzed.
+        /// If not set, it will analyze all sizes.
+        /// </summary>
+        public int FileSizeInKilobytes { get; set; } = -1;
```

Comment:
>-1 [](http://example.com/codeflow?start=55&length=2)

Remove this. Did you author a test for the validation command that this is honored?